modules/assim.batch/R/pda.emulator.R

```diff
@@ -26,9 +26,6 @@ pda.emulator <- function(settings, params.id=NULL, param.names=NULL, prior.id=NU
                   prior.id=prior.id, chain=chain, iter=iter, adapt=adapt, 
                   adj.min=adj.min, ar.target=ar.target, jvar=jvar, n.knot=n.knot, burnin=burnin)
```

Comment:
I agree that the elements required in the settings list need to be documented. I'd be fine with documenting the HIGH LEVEL elements required and then pointing to the Gitbook for details about the elements within each list.